arrow/arrow.py

```diff
@@ -72,7 +72,7 @@ class Arrow(object):
     _SECS_PER_DAY = float(60 * 60 * 24)
     _SECS_PER_WEEK = float(60 * 60 * 24 * 7)
     _SECS_PER_MONTH = float(60 * 60 * 24 * 30.5)
-    _SECS_PER_YEAR = float(60 * 60 * 24 * 365.25)
+    _SECS_PER_YEAR = float(60 * 60 * 24 * 365)
```

Comment:
Should we put a comment similar to the weeks so that we remember to revisit year humanization during leap years?